wibu_catalog/models.py

```diff
@@ -1,8 +1,14 @@
 # wibu_catalog/models.py
+import uuid
 from cloudinary.models import CloudinaryField
 from django.db import models
 from django.urls import reverse
 from django.utils.translation import gettext_lazy as _
+from django.urls import reverse
+from django.db.models.signals import post_save
+from django.dispatch import receiver
+from django.contrib.auth.hashers import make_password
+from django.contrib.auth.hashers import check_password
```

Comment:
gộp 1 dòng import đc nè e
```suggestion
from django.contrib.auth.hashers import make_password, check_password
```